### PR TITLE
Update 02_connect_to_cluster.mdx

### PR DESCRIPTION
**Pull request by theadamwright**
Added Best Practice Recommendation for SSL Mode

## What Changed?

Added `sslmode` section 

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [X ] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content

### CODE REVIEW

File: product_docs/docs/edbcloud/beta/using_cluster/02_connect_to_cluster.mdx [hunk 1]
```diff
@@ -12,4 +14,14 @@ To connect to your cluster using `psql`:
 To connect to your cluster using a client other than `psql`:
 1. Go to the [**Clusters**](https://portal.edbcloud.com/clusters) page.
 2. Select the name of your cluster. 
-3. Select the **Connect** tab. You can review and copy all the relevant information you need from this screen except for the `edb_admin` user password. Please consult the client driver documentation for the connection string format the driver uses.  
+3. Select the **Connect** tab. You can review and copy all the relevant information you need from this screen except for the `edb_admin` user password. Please consult the client driver documentation for the connection string format the driver uses.
+
+## Best Practice Recommendation - SSL Mode
+
+Different clients can have different default SSL modes. For example, `psql` defaults to `prefer`, which means the client will attempt to establish a SSL connection but fall back to non-SSL if the server does not support it. In the `psql` example provided by EDB in the **Quick Connect** field, SSL mode is explicitly set to `require`, which means the client will attempt an SSL connection and fail if the connection to the server can't be encrypted.
```
Comment (ba-edb) [inline product_docs/docs/edbcloud/beta/using_cluster/02_connect_to_cluster.mdx:21]: Can we do some sort of wordsmithing to note that we don't actually support SSL, that we support TLS (v1.2+), but PG still talks about SSL for historical reasons?

File: product_docs/docs/edbcloud/beta/using_cluster/02_connect_to_cluster.mdx [hunk 2]
```diff
@@ -12,4 +14,14 @@ To connect to your cluster using `psql`:
...
+
+## Best Practice Recommendation - SSL Mode
+
+Different clients can have different default SSL modes. For example, `psql` defaults to `prefer`, which means the client will attempt to establish a SSL connection but fall back to non-SSL if the server does not support it. In the `psql` example provided by EDB in the **Quick Connect** field, SSL mode is explicitly set to `require`, which means the client will attempt an SSL connection and fail if the connection to the server can't be encrypted. 
+
+For public connections and in most environments, EDB recommends setting SSL mode to `verify-full`. This ensures that you connect to the server you specified and that the connection is encrypted. EDB Cloud generates *letsEncrypt* certificates, a trusted CA. Therefore, your client machine will need a CA Cert for Let's Encrypt[https://letsencrypt.org/certificates/]. Once the CA cert is in place on your client machine, add `PGSSLROOTCERT` and set `PGSSLMODE` to `verify-full` in your connection string to verify the certificate:
```
Comment (ba-edb) [inline product_docs/docs/edbcloud/beta/using_cluster/02_connect_to_cluster.mdx:23]:
```suggestion
For public connections and in most environments, EDB recommends setting SSL mode to `verify-full`. This ensures that you connect to the server you specified and that the connection is encrypted. EDB Cloud generates certificates with LetsEncrypt a widely trusted certificate authority. Your client machine may already have a bundled CA certificate for LetsEncrypt, e.g., at `/etc/ssl/certs/ca-certificates.crt` or `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem`. If it doesn't, your client machine will need a [CA certificate for Let's Encrypt](https://letsencrypt.org/certificates/). Once the CA certificate is in place on your client machine, configure the `sslrootcert` parameter to its location and set the `sslmode` parameter to `verify-full` to verify the certificate to fully validate the connection:
```

File: product_docs/docs/edbcloud/beta/using_cluster/02_connect_to_cluster.mdx (hunk 1 above)
Comment (theadamwright) [reply product_docs/docs/edbcloud/beta/using_cluster/02_connect_to_cluster.mdx]: @ba-edb Let me know if you agree with the latest changes.

**Comment by ba-edb** [comment]
LGTM

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**